### ISSUE TITLE
Proposal: new mutation status for mutants that are covered by a test that fails even before mutation

### ISSUE THREAD

**Issue by Mobrockers** [stryker-mutator/mutation-testing-elements#158]
In stryker-net we have coverage analysis so we can determine which mutants are covered by which tests. Currently we do not allow a mutation testrun if you have failing unit tests because this would mark every mutant as killed. With coverage analysis we can disable all unit tests that are failing while also marking and skipping all mutants covered by those failing unit tests. We would like to have a new status in the schema to reflect a mutant that is covered by a test but for which we cannot determine if it's killed or not due to the covering unit test not working.

**Comment by Mobrockers** [comment]
@nicojs thoughts?

**Comment by nicojs** [comment]
Can we use the `ignored` status with this and reflect the reason in a (new) reason field?

**Comment by Mobrockers** [comment]
Yes that's a good idea, we can fill the reason field already for others already as well.

**Comment by nicojs** [comment]
Great, let's do that than. Btw, I would suggest to have a compiler flag to disable this feature in Stryker, since you might be fooled by the mutation score.

**Comment by Mobrockers** [comment]
Proposal: Ability to tag mutants with traits such as:

'Slow', 'Static context', 'Initial testrun failed' etc instead of new mutant statusses
